app/src/main/java/com/github/se/eventradar/ui/login/LoginScreen.kt

```diff
@@ -46,155 +47,156 @@ import com.github.se.eventradar.ui.navigation.Route
 
-  val intent =
-      AuthUI.getInstance()
-          .createSignInIntentBuilder()
-          .setIsSmartLockEnabled(false)
-          .setAvailableProviders(providers)
-          .build()
+    val openErrorDialog = remember { mutableStateOf(false) }
 
-  Column(
-      modifier = Modifier.fillMaxSize().testTag("loginScreen"),
-      verticalArrangement = Arrangement.Center,
-      horizontalAlignment = Alignment.CenterHorizontally,
-  ) {
-    ConstraintLayout {
-      val (titleRow, signInButton, signUpRow) = createRefs()
-      Row(
-          modifier =
-              Modifier.constrainAs(ref = titleRow, { top.linkTo(parent.top, margin = 16.dp) }),
-          horizontalArrangement = Arrangement.Center,
-          verticalAlignment = Alignment.CenterVertically) {
-            Image(
-                painter = painterResource(R.drawable.event_radar_logo),
-                contentDescription = "Logo",
-                modifier = Modifier.padding(1.dp).width(43.dp).height(68.dp).testTag("logo"),
+    val launcher = rememberLauncherForActivityResult(
+        contract = FirebaseAuthUIActivityResultContract(),
+        onResult = { result ->
+            if (result.resultCode == Activity.RESULT_OK) navigationActions.navController.navigate(
+                Route.OVERVIEW
             )
-            Text(
-                text = "Event Radar",
-                modifier = Modifier.width(253.dp).height(55.dp).testTag("loginTitle"),
-                style =
-                    TextStyle(
-                        fontSize = 40.sp,
+            else openErrorDialog.value = true
+        })
+
+    ErrorDialogBox(openErrorDialog = openErrorDialog)
+
+    val providers = arrayListOf(
+        AuthUI.IdpConfig.GoogleBuilder().build(),
+    )
+
+    val intent = AuthUI.getInstance().createSignInIntentBuilder().setIsSmartLockEnabled(false)
+        .setAvailableProviders(providers).build()
+
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .testTag("loginScreen"),
+        verticalArrangement = Arrangement.Center,
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        ConstraintLayout {
+            val (titleRow, signInButton, signUpRow) = createRefs()
+            Row(
+                modifier = Modifier.constrainAs(ref = titleRow) {
+                    top.linkTo(
+                        parent.top,
+                        margin = 16.dp
+                    )
+                },
+                horizontalArrangement = Arrangement.Center,
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Image(
+                    painter = painterResource(R.drawable.event_radar_logo),
+                    contentDescription = "Logo",
+                    modifier = Modifier
+                        .padding(1.dp)
+                        .width(300.dp)
+                        .height(100.dp)
+                        .testTag("logo"),
+                )
```

Comment:
I agree, nice change for the logo, just a little reminder to make sure the tests for Login still work with the Image in case it checks for "Event Radar" as Text :)

Reply:
I agree, changed it in both LoginScreen and SignUpScreen 👍